File: dotainfo-ios-app/DotaInfo/ContentView.swift
import DotaInfoShared
import SwiftUI

struct ContentView: View {
    let greeting = "Hello, \(Meta.shared.platform)!"

    var body: some View {
        Text(greeting)
            .padding()
    }
}

struct ContentView_Previews: PreviewProvider {
    static var previews: some View {
        ContentView()
    }
}

File: dotainfo-android-app/src/main/kotlin/dev/gressier/dotainfo/android/ui/ContentView.kt
package dev.gressier.dotainfo.android.ui

import androidx.compose.foundation.layout.Box
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.material.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.tooling.preview.Preview
import dev.gressier.dotainfo.shared.Meta

@Composable
fun ContentView() {
    val greeting = "Hello, ${Meta.platform}!"

    Box(
        Modifier.fillMaxSize(),
        contentAlignment = Alignment.Center,
    ) {
        Text(greeting)
    }
}

@Preview
@Composable
fun ContentView_Preview() {
    ContentView()
}